### PR TITLE
feat(ui): .NET stack trace improvements

### PR DESCRIPTION
**Pull request by matejminar**
### Before
image

### After
image

- hide pathname in title
- fix regression of path input
- add tooltip with full path to path input
- remove IconReturn
- create a mapping function for stack trace platform icon

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/interfaces/frame/utils.tsx [hunk 1]
```diff
@@ -62,3 +62,7 @@ export function getFrameHint(frame: Frame) {
 
   return [null, null];
 }
+
+export function isCsharp(platform: string) {
```
Comment (bruno-garcia) [inline src/sentry/static/sentry/app/components/events/interfaces/frame/utils.tsx:66]: what about naming this function `isDotnet` ?

platform=csharp actually represents `.NET` and can be `F#`, `VB` or any language targeting CLS (the Common Language Specification).

Maybe this ^ as a comment inside the function

**Review by bruno-garcia — COMMENTED** [review]
This looks great!

**Review by Tyrrrz — APPROVED** [review]

File: src/sentry/static/sentry/app/components/events/interfaces/frame/utils.tsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/components/events/interfaces/frame/utils.tsx]: sounds good, you got it 👍

**Review by markstory — APPROVED** [review]
Looks good to me.

**Review by dashed — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]

**Pull request merged**